modules/card/react/lib/Card.tsx

```diff
@@ -44,7 +45,9 @@ export interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
   height?: number | string;
 }
 
-const Box = styled('div')<CardProps>(
+const Box = styled('div', {
+  shouldForwardProp: prop => isPropValid(prop) && prop !== 'width' && prop !== 'height',
```

Comment:
maybe without negation
```
 isPropValid(prop) || prop === 'width' || prop === 'height',
```

Reply:
Not sure I follow - `prop === 'width'` would return true for `width`, which would mean it does get forwarded to the html element (not what we want)

Reply:
@NicholasBoll Oh Okay, my bad! I thought we want to pass it